src/Http/Middleware/CsrfProtectionMiddleware.php

```diff
@@ -193,6 +193,26 @@ public function createToken(): string
         return $value . hash_hmac('sha1', $value, Security::getSalt());
     }
 
+    /**
+     * Verify CSRF token.
```

Comment:
Can we add a little more explanation on the verification requirements? We want to document the hardened token change a little more than just a couple lines of code changing.